### ISSUE TITLE
tol_induced_subtree errors with v3

### ISSUE THREAD

**Issue by dwinter** [ropensci/rotl#74]
Using [the example from the docs](https://github.com/OpenTreeOfLife/germinator/wiki/Synthetic-tree-API-v3#induced_subtree) we get a `neo4j` error about casting strings (?) when using `rotl`

```r
test <- c(292466, 267845, 666104, 316878, 102710)
tol_induced_subtree(ott_ids=test)
```

```
Could not convert!BadInputExceptionorg.neo4j.server.rest.repr.BadInputExceptionlist("org.neo4j.server.rest.repr.RepresentationFormat.convertString(RepresentationFormat.java:330)", "org.neo4j.server.rest.repr.RepresentationFormat$1.convertString(RepresentationFormat.java:164)", "org.neo4j.server.plugins.ParameterList.getString(ParameterList.java:86)", "org.neo4j.server.plugins.StringTypeCaster.get(StringTypeCaster.java:30)", "org.neo4j.server.plugins.ParameterExtractor.extract(ParameterExtractor.java:45)", "org.neo4j.server.plugins.PluginMethod.invoke(PluginMethod.java:53)", 
```

The same examples works fine with `curl` from command line, so seems like it is something we are doing

```sh
curl -X POST https://api.opentreeoflife.org/v3/tree_of_life/induced_subtree -H "content-type:application/json" -d {"ott_ids":[292466, 267845, 666104, 316878, 102710]}'
```

```
{
  "newick" : "(((Cinclus_ott26784 ....
}
```

**Comment by josephwb** [comment]
Hmm. I'll take a look.

**Comment by josephwb** [comment]
Ack, it looks like all of my edits were overwritten!

**Comment by josephwb** [comment]
Ok, found the issue. Not sure what to do about all of my deleted edits, though...

**Comment by josephwb** [comment]
Fixed with 88e0f6cb03721725967b8c2395a5ece783a690df.
